mscore/selectnotedialog.cpp

```diff
@@ -49,7 +49,8 @@ SelectNoteDialog::SelectNoteDialog(const Note* _n, QWidget* parent)
       pitch->setText(n->tpcUserName());
       string->setText(QString::number(n->string()+1));
       type->setText(n->noteTypeUserName());
-      duration->setText(n->chord()->durationUserName());
+      durationType->setText(n->chord()->durationType().durationTypeUserName() + QString(" Note"));
```

Comment:
QString(" Note") makes this a not-translatable thing

Reply:
I agree, how would you go about fixing this? I'm not familiar with musescore's localization

Reply:
`... + " " + tr("Note`));` `
or 
`durationType->setText(tr(%1 Note").arg(n->chord()->durationType().durationTypeUserName());

Reply:
@Jojo-Schmitz Thanks! I didn't know about the `tr` function, updated